beacon-chain/main.go

```diff
@@ -141,7 +146,18 @@ func main() {
 				log.WithError(err).Error("Failed to configuring logging to disk.")
 			}
 		}
-
+		if ctx.IsSet(cmd.ChainConfigFileFlag.Name) {
+			chainConfigFileName := ctx.String(cmd.ChainConfigFileFlag.Name)
+			yamlFile, err := ioutil.ReadFile(chainConfigFileName)
+			if err != nil {
+				log.WithError(err).Error("Failed to read chain config file.")
+			}
+			conf := params.BeaconConfig()
+			if err := yaml.Unmarshal(yamlFile, conf); err != nil {
+				log.WithError(err).Error("Failed to parse chain config yaml file.")
```

Comment:
```suggestion
				log.WithError(err).Error("Failed to parse chain config YAML file.")
```
YAML is acronym, so should be in all caps.